etc/rose-rug-brief-tour/suite.rc

```diff
@@ -1,14 +1,10 @@
 #!jinja2
 [cylc]
-    # Begin Rose required event hook block
     UTC mode = True # Ignore DST
-    [[event hooks]]
-        timeout handler = rose suite-hook --mail --shutdown
-        timeout = P3D
-    # End Rose required event hook block
+    abort if any task fails = True
 [scheduling]
-    initial cycle point = 20130101T0000Z # The first cycle of the suite is at 00:00 on Jan 1 2013
-    final cycle point = 20130102T0000Z # The last cycle of the suite is at 00:00 on Jan 2 2013
```

Comment:
This "last cylc of..." entry was very useful to have in as it highlights that it isn't a stop at that point it's a that's the last cycle that will be carried out - something we're used to but confuses new users.